components/clarity-jupyter-kernel/Cargo.toml

```diff
@@ -8,21 +8,25 @@ readme = "README.md"
 license = "GPL-3.0-only"
 keywords = ["blockchain", "clarity", "smart-contract", "jupyter", "notebook"]
 repository = "https://github.com/hirosystems/clarinet/components/clarity-jupyter-kernel"
-categories = ["command-line-utilities", "development-tools", "development-tools::build-utils"]
+categories = [
+  "command-line-utilities",
+  "development-tools",
+  "development-tools::build-utils",
+]
 
 [dependencies]
 clarity_repl = { package = "clarity-repl", path = "../clarity-repl" }
 lazy_static = "1.4.0"
-regex = "1.3.4"
+regex = "1.7"
 sha2 = "0.8.1"
 sha3 = "0.8.2"
 ripemd160 = "0.8.0"
-json = { version = "0.11.15" } # should use serde_json instead
-failure = { version = "0.1.5", default-features = false, features = [ "std" ] }
+json = { version = "0.11.15" }                                                # should use serde_json instead
```

Comment:
auto format from the toml extension i have.. do we like it 🤔

Reply:
No strong opinion on my end.